### ISSUE TITLE
Request: Please consider including emacs

### ISSUE THREAD

**Issue by AnonVendetta** [Zackptg5/Cross-Compiled-Binaries-Android#11]
Honestly, I havent found a good, recent version of emacs for Android. I see that nano and vim are available for download. But emacs, one of the longstanding best editors out there, isnt available.

I'm sure I'm not the only one who would want to see this. I understand if the ccbins maintainers are vim/nano fanboys, but at least give us a 3rd choice, let the user decide according to their preference. Choice is always good.

Thanks for the consideration!

**Comment by Zackptg5** [comment]
I'll give it a look over and let you know how it goes, thanks

**Comment by AnonVendetta** [comment]
I've recently discovered that Termux has an up to date Emacs package. There is also an app called Emacs on Play Store, but the downside is that you cant get it fully working without disabling Android's position independent executables security implementation. That app is also very old.

I'm looking forward to seeing a native version of Emacs, thanks!

**Comment by Zackptg5** [comment]
Gave it a shot and looks like there's a lot of dependencies and such I'll need to work out. I'll chip away at it when I get the chance but no guarantees :/

**Comment by AnonVendetta** [comment]
Lots of depencies? Like what, exactly? Do nano and vim lack dependencies (i.e. they're standalone)? Or maybe you just mean that compiling from source has lots of dependencies, but the resulting binary should be standalone?

**Comment by Zackptg5** [comment]
Compiling from source. All binaries I put here are static linked meaning they're standalone

**Comment by Zackptg5** [comment]
Unfortunately, I couldn't get it to cross compile. It compiles binaries that it uses to generate other binaries which makes cross compiling not possible. I tried to compile x64 copies of the binaries in question and copy them over during the build but it happens during the make process so I couldn't get it to work. If you want to give it a shot, I did need to make a patch and use the --without-all configured flag. I attached the patch if you're interested but I won't be able to add it currently. Sorry :/
[emacs.zip](https://github.com/Zackptg5/Cross-Compiled-Binaries-Android/files/5221173/emacs.zip)
